bitwarden_license/src/app/providers/clients/add-organization.component.ts

```diff
@@ -48,7 +52,12 @@ export class AddOrganizationComponent implements OnInit {
 
         this.provider = await this.userService.getProvider(this.providerId);
 
-        this.organizations = (await this.userService.getAllOrganizations()).filter(p => p.providerId == null);
+        const candidateOrgs = (await this.userService.getAllOrganizations()).filter(o => o.providerId == null);
+        const allowedOrgsIds = await Promise.all(candidateOrgs.map(o => this.apiService.getOrganization(o.id))).then(orgs =>
```

Comment:
Just a thought - is this `load` method call behind a spinner since we're potentially making _N_ api calls back to back? Could potentially take some time if there are a lot of `candidateOrgs`

Reply:
It is. I thought about adding the org plan to the info in userService, but it kind of breaks our paradigm of keeping that model reserved for user-specific data.

in the end I think the separation of concerns is better and if we run into issues with loading times, the better solution is to make a specialized api call for this.

Reply:
Seems good to me!